AppCenterDataStorage/AppCenterDataStorage/MSDataStore.h

```diff
@@ -16,23 +16,18 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
-/**
- * Error domain for Storage.
- */
-static NSString *const kMSDataStorageErrorDomain = @"MSDataStorageErrorDomain";
-
 /**
  * User partition.
  * An authenticated user can read/write documents in this partition.
  */
-static NSString *const MSDataStoreUserDocumentsPartition = @"user";
+static NSString *const kMSDataStoreUserDocumentsPartition = @"user";
```

Comment:
Those are meant to be public. Isn't the `kMS` prefix used for private constants only?